src/ordering.c

```diff
@@ -450,6 +488,28 @@ enum local_subsection get_local_subsection(const struct policy_node *node)
 	}
 }
 
+/*
+ * Treat the following as the same section:
+ *       foo_t and foo_t
+ *       foo_t and foo_r
+ *       foo   and foo_t
+ */
+static int is_same_section(const char *first_section_name, const char *second_section_name)
+{
+	size_t first_length = strlen(first_section_name);
+	size_t second_length = strlen(second_section_name);
+
+
+	if (first_length >= 3 && (first_section_name[first_length-1] == 't' || first_section_name[first_length-1] == 'r') && first_section_name[first_length-2] == '_') {
```

Comment:
Can you please break this into multiple lines so that the boolean logic is clearer for readers?  Same with "second_length" equivalent.